crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -901,6 +901,10 @@ where
             ..
         } = request;
 
+        let kind = match to {
+            Some(tx_kind) => tx_kind,
+            None => RpcTransactionKind::Create,
+        };
```

Comment:
i think `RpcTransactionKind` is the same as `alloy_primitives::TxKind` so we should be able to just do `to.unwrap_or(TxKind::Create)`?

Reply:
good point, done